tests/test_match_labels.py

```diff
@@ -3,29 +3,51 @@
 import numpy as np
 import biapol_utilities as biau
 
+
 def test_match_labels():
 
     labels_x = np.asarray([1, 1, 1, 0, 0, 2, 2, 4, 4, 4, 0], dtype=np.uint8)
     labels_y = labels_x * 2
 
-    labels_y_matched = biau.label.match_labels(labels_x, labels_y)
+    m_method1 = biau.label.max_similarity
+    m_method2 = biau.label.gale_shapley
 
-    # calculate number of overlapping matches
-    match = [x == y for x,y in zip(labels_x, labels_y_matched)]
+    labels_y_matched1 = biau.label.match_labels(labels_x, labels_y,
+                                                matching_method=m_method1)
 
-    assert(all(match))
+    assert np.array_equal(labels_y_matched1, labels_x)
 
 
 def test_match_labels_2():
     labels_x = np.asarray([1, 1, 6, 0, 0, 3, 3, 4, 4, 4, 0], dtype=np.uint8)
     labels_y = np.asarray([1, 1, 3, 0, 0, 2, 2, 5, 5, 5, 4], dtype=np.uint8)
 
-    reference_y_matched = np.asarray([1, 1, 6, 0, 0, 3, 3, 4, 4, 4, 7], dtype=np.uint8)
+    reference_y_matched = np.asarray([1, 1, 6, 0, 0, 3, 3, 4, 4, 4, 7],
+                                     dtype=np.uint8)
+
+    m_method = biau.label.max_similarity
+    labels_y_matched = biau.label.match_labels(labels_x, labels_y,
+                                               matching_method=m_method)
```

Comment:
Would it make sense to add a second test where `*order*` is specifed?

Reply:
Yes and no. 
I added a line to the matching algorithm that force-sets the order to `descending`, because this should be clearly specificed here. 
No, because changing this would mean to match up labels with the lowest similarity. If a metric takes low values for high similarity (e.g. Euclidian distance between centroids), then the distance matrix should rather be inverted as it is written in the docstring of `match_labels(...)`